src/state/store.ts

```diff
@@ -47,6 +45,23 @@ export const clearState = (opts?: { excludePreferences?: boolean }) => {
   store.dispatch(apiSlices.stakingDataApi.util.resetApiState())
 }
 
+const actionSanitizer = (action: any) => {
+  const blackList = [
+    'asset/setAssets',
+    'assetApi/executeQuery/fulfilled',
+    'marketData/setMarketData',
+    'marketData/setPriceHistory',
+  ]
+  return blackList.includes(action.type)
+    ? {
+        ...action,
+        payload: 'see actionSanitizer in store.ts',
+      }
+    : action
+}
+
+const stateSanitizer = (state: any) => ({ ...state, assets: 'see stateSanitizer in store.ts' })
```

Comment:
blacklist the 6000+ assets and their ids from serialization

Reply:
Very nice.